### ISSUE TITLE
Don't generate the storage/docs/api-docs.json file

### ISSUE THREAD

**Issue by aureliomachados** [slampenny/Swaggervel#68]
When i access localhost:8000/docs: Cannot find /home/aurelio/laravel/laravelgenerator/storage/docs/api-docs.json

I made all configurations, step by step, but when i generate some resource the api-docs.json is not created.

obs: my storage directory is free to write and read.

**Comment by GNum116** [comment]
Me too

**Comment by krisbrkrunch** [comment]
Ive not been able to get any kind of automatic generation going only option for me was to do manual annotations in the php files then run command line ./vendor/bin/swagger -o /storage/docs to get the json generated.

**Comment by mhetreramesh** [comment]
I am also facing  same issue! Does anyone has facing this issue.. Any solution?

**Comment by 3tgame** [comment]
add some notations，visit the http:localhost:8000/api-docs  url

**Comment by 3tgame** [comment]
me too，I use Laravel 5.1 and swaggervel dev-master

**Comment by rilefitra** [comment]
I am also facing same issue

**Comment by ansezz** [comment]
I am also facing same issue

**Comment by hillelcoren** [comment]
I was able to fix this by setting APP_DEBUG=true in my .env file

**Comment by o0JoeCool0o** [comment]
FYI I created an automatic annotation generator called swaggertater that scans your output of php artisan route:list and then scans your classes for input variables and generates some annotations! Its written in objective c and currently only works on mac. I use it on my dev environment and then I upload the annotations.php file to my prod server. https://github.com/krisbray/SwaggerTater  and see my updated fork of swaggervel here https://github.com/krisbray/Swaggervel

**Comment by w0rd-driven** [comment]
The route defined by the config line https://github.com/slampenny/Swaggervel/blob/master/src/config/swaggervel.php#L29 is what generates the `storage/docs/api-docs.json` file. You'll need to make some annotations as @3tgame mentions.

I honestly found this a little confusing so my route definitions are as follows:
```
    /*
      |--------------------------------------------------------------------------
      | Relative path to access parsed swagger annotations.
      |--------------------------------------------------------------------------
    */
    'doc-route' => 'docs-json',

    /*
      |--------------------------------------------------------------------------
      | Relative path to access swagger ui.
      |--------------------------------------------------------------------------
    */
    'api-docs-route' => 'docs',
```
Going to `/docs` gets the Swagger UI I normally expect. Going to `/docs-json` gives me the JSON the UI consumes. This is one area that confused me initially and the naming is a little unfortunate as I don't really have a good permanent alternative. Changing the route somewhat signifies intent.

**Comment by o0JoeCool0o** [comment]
Yes I use SwaggerTater above to make those annotations now automatically. I also set the autogen to true so after SwaggerTater is finished it generates the api-docs.js file and I have even edited my Swaggervel instance to run it on load of the docs page so I have FULLY automated api page now all of my functions just show up with parameters any time I add or change them I dont have to do any edits.

**Comment by thiernosow219** [comment]
bonjour a vous sil vous plait j'arrive pas a documenter mes api avec swagger avez vous uune solution
